### PR TITLE
Create What is WCS section

### PR DESCRIPTION
**Pull request by Kuboosh21**
New card for What is West Coast Swing

Couple of issues I need help with:
* The width of the card is weird, and I had to set a very specific max-width to match the Upcoming Events section. Would like to make this cleaner
* Couldn't figure out bullet points with Typography tags
* double check video width is nice for all sizes

image

image

image

image

### CODE REVIEW

File: web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.styles.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import theme from "common/theme";
+
+const whatIsWcsStyles = {
+  whatIsWcsStylesContainer: {
+    padding: "24px",
+    maxWidth: "1152px", // to match Upcoming Events. Halp me here plz
```
Comment (jesseez) [inline web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.styles.js:6]: https://github.com/doyourownswing/doyourownswing.github.io/blob/4dbd37544e16516f7d6eca458e6cc535dca4aa6c/web/src/components/home/subcomponents/upcoming_events/upcoming_events.js#L80C8-L80C17

Use Container, inside a Box (Box is the whole section element, and Container holds the content and has the maxWidth styling)

File: web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+              referrerpolicy="strict-origin-when-cross-origin"
+              allowfullscreen
+            ></iframe>
+          </Box>
+          <Stack
+            sx={whatIsWcsStyles.descriptionContainer}
+            spacing={{ xs: 1, md: 2 }}
```
Comment (jesseez) [inline web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js:27]: Nit: I've been moving stuff like this into the styles object too

File: web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
-  return <div>What is west coast swing</div>;
+  return (
+    <Box sx={whatIsWcsStyles.whatIsWcsStylesContainer}>
+      <Typography variant="h4" sx={whatIsWcsStyles.header}>
+        What is West Coast Swing?
+      </Typography>
+      <Box sx={whatIsWcsStyles.itemsContainer}>
+        <Stack sx={whatIsWcsStyles.twoColumnContent}>
+          <Box sx={whatIsWcsStyles.videoContainer}>
+            <iframe
```
Comment (jesseez) [inline web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js:14]: Might be nice to extract this into it's own component

**Review by jesseez — COMMENTED** [review]
Overall LGTM. Will still try to revisit content. Also use the Container element

File: web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js (hunk 2 above)
Comment (Kuboosh21) [reply web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js:27]: I couldn't get this specific property into the styles object, it wouldn't translate to an actual css property for some reason

File: web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js (hunk 3 above)
Comment (Kuboosh21) [reply web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js]: done

File: web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.styles.js (hunk 1 above)
Comment (Kuboosh21) [reply web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.styles.js]: thanks - updated

**Review by jesseez — APPROVED** [review]
Chatted offline, the way we are handling messages is kinda funky. Let's take this as an AI and do it as a fast follow. Thanks!

**Pull request merged**